app/src/main/res/xml/preferences.xml

```diff
@@ -245,6 +245,20 @@
             android:title="@string/settings_proxy" />
     </PreferenceScreen>
 
+    <PreferenceScreen
+        search:ignore="true"
+        android:key="pref_category_android_auto"
+        android:title="@string/settings_android_auto">
+
+        <CheckBoxPreference
+            android:disableDependentsState="true"
+            android:key="show_station_icons_in_hud_browser"
```

Comment:
Should this setting be set to "false" by default since it may cause incompatibilities?
The string of dependents state is redundant.